source/index.js

```diff
@@ -1,67 +1,76 @@
 const {app, BrowserWindow} = require('electron');
-  const path = require('path')
-  const url = require('url')
-  
-  // Keep a global reference of the window object, if you don't, the window will
-  // be closed automatically when the JavaScript object is garbage collected.
-  let mainWindow
-  
-  function createWindow () {
-    // Create the browser window.
-    mainWindow = new BrowserWindow({
-      titleBarStyle: 'hidden', 
-      resizable: true, 
-      frame: false, 
-      minWidth: 720,
-      maxWidth: 720,
-      width: 720,
-      height: 930,
-      webPreferences: {
-        devTools: true
-      },
-      icon: path.join(__dirname, '/assets/img/logo_normal.png')
-    })
-  
-    // and load the index.html of the app.
-    mainWindow.loadURL(url.format({
-      pathname: path.join(__dirname, 'index.html'),
-      protocol: 'file:',
-      slashes: true
-    }))
-  
-    // Open the DevTools.
-    // mainWindow.webContents.openDevTools()
-  
-    // Emitted when the window is closed.
-    mainWindow.on('closed', function () {
-      // Dereference the window object, usually you would store windows
-      // in an array if your app supports multi windows, this is the time
-      // when you should delete the corresponding element.
-      mainWindow = null
-    })
-  }
-  
-  // This method will be called when Electron has finished
-  // initialization and is ready to create browser windows.
-  // Some APIs can only be used after this event occurs.
-  app.on('ready', createWindow)
-  
-  // Quit when all windows are closed.
-  app.on('window-all-closed', () => {
-    // On macOS it is common for applications and their menu bar
-    // to stay active until the user quits explicitly with Cmd + Q
-    if (process.platform !== 'darwin') {
-      app.quit()
-    }
+const path = require('path')
+const url = require('url')
+const {FileWatcher} = require('./filewatcher');
+const PORT = 4455;
+const FILE_PATH = '__temp-tail.txt';
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let mainWindow
+let fileWatcher;
+
+function createWindow () {
+  // Create the browser window.
+  mainWindow = new BrowserWindow({
+    titleBarStyle: 'hidden', 
+    resizable: true, 
+    frame: false, 
+    minWidth: 720,
+    maxWidth: 720,
+    width: 720,
+    height: 930,
+    webPreferences: {
+      devTools: true
+    },
+    icon: path.join(__dirname, '/assets/img/logo_normal.png')
   })
-  
-  app.on('activate', () => {
-    // On macOS it's common to re-create a window in the app when the
-    // dock icon is clicked and there are no other windows open.
-    if (wimainWindown === null) {
```

Comment:
This is the misspelled variable name - it could actually potentially cause a JS crash since the variable is never initialized anywhere.